src/content/perf/faq.md

```diff
@@ -94,10 +94,10 @@ about evaluating and debugging Flutter's performance.
 * How do I see the breakdown of the Flutter engine size?
   * Visit the [binary size dashboard][], and replace the git
     hash in the URL with a recent commit hash from
-    [GitHub engine repository commits][].
+    [Flutter's GitHub commits][].
 
 [binary size dashboard]: https://storage.googleapis.com/flutter_infra_release/flutter/241c87ad800beeab545ab867354d4683d5bfb6ce/android-arm-release/sizes/index.html
-[GitHub engine repository commits]: {{site.repo.engine}}/commits
+[Flutter's repository commits]: {{site.repo.flutter}}/commits
```

Comment:
@jtmcdole Could you confirm this is correct?

Reply:
the binary size looks wrong (the dates are 2021?).

flutter/flutter/commits is right: https://github.com/flutter/flutter/commits/master/

Reply:
master -> main works, too and we far prefer it.

Reply:
Updated to {{site.repo.flutter}}/commits/main!